TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/CaptureProxy.java

```diff
@@ -191,19 +195,20 @@ static Parameters parseArgs(String[] args) {
 
     @SneakyThrows
     protected static Settings getSettings(@NonNull String configFile) {
```

Comment:
Can we add a unit test since we need coverage + we are touching previously untested coded that apparently had bugs?

Reply:
Parsing tests added.  Notice that we get some tests here from the E2E tests.  We just weren't setting compound configurations in the proxy invocation that the E2E scripts were using.  Once I turned that on, things went sideways.